.circleci/db-migration.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+#
+# Copyright 2018-2022 contributors to the Marquez project
+# SPDX-License-Identifier: Apache-2.0
+#
+# Usage: $ ./db-migration.sh
```

Comment:
A descriptive comment on this file will tell readers the intention of this script - it's a bit hard to parse by just reading the bash